assets/queries/k8s/protect_kernel_defaults_is_not_set_to_false/metadata.json

```
@@ -0,0 +1,11 @@
+{
+    "id": "6cf42c97-facd-4fda-b8af-ea4529123355",
+    "queryName": "Protect Kernel Defaults Is Not Set To False",
```

```suggestion
       "queryName": "Kubelet Protect Kernel Defaults Set To False",
```

Please, also update the query folder name and the `positive_expected_result.json`